common/tools/dev-tool/src/commands/migrate.ts

```diff
@@ -354,7 +354,7 @@ async function runMigrations(pending: Migration[], project: ProjectInfo): Promis
 async function onMigrationSuccess(
   project: ProjectInfo,
   migration: Migration,
-  quiet: boolean = false
+  quiet: boolean = false // eslint-disable-line @typescript-eslint/no-inferrable-types
```

Comment:
Why can't we omit the type annotation?

Reply:
because the inferred type will only false only?

Reply:
This is a parameter type. It's a typescript-eslint bug that has been fixed in latest version.  I will remove these when upgrading those dependencies.